src/app/common/components/presentation/check-box/check-box.component.html

```diff
@@ -1,3 +1 @@
-<p>
-  DRIVER-CHECKBOX
-</p>
+<mat-checkbox [color]="color">{{label}}</mat-checkbox>
```

Comment:
checkbox needs to store a model that can be retrieved and have an event emitter for changes.